src/main/kotlin/blackjack/domain/player/Players.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack.domain.player
+
+import blackjack.domain.card.Cards
+
+class Players(val value: List<Player>) {
+    fun receiveCard(getCards: () -> Cards) {
+        val cards = getCards()
+        value.forEach { player -> player.initCards(cards) }
```

Comment:
```suggestion
        value.forEach { it.initCards(cards) }
```

it을 쓰셔도 괜찮을 것 같습니다  :)